californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfigDefaults.java

```diff
@@ -76,7 +76,7 @@ public static void setDefaults(final NetworkConfig config) {
 		config.setInt(NetworkConfig.Keys.PREFERRED_BLOCK_SIZE, 512);
 		config.setInt(NetworkConfig.Keys.MAX_MESSAGE_SIZE, 1024);
 		config.setInt(NetworkConfig.Keys.MAX_RESOURCE_BODY_SIZE, DEFAULT_MAX_RESOURCE_BODY_SIZE);
-		config.setInt(NetworkConfig.Keys.BLOCKWISE_STATUS_LIFETIME, 10 * 60 * 1000); // ms
+		config.setInt(NetworkConfig.Keys.BLOCKWISE_STATUS_LIFETIME, DEFAULT_BLOCKWISE_STATUS_LIFETIME); // ms
```

Comment:
Reason for changing the default from 10 minutes to 30 seconds?
A device restart or a lost DTLS session may exceed 30 seconds, so would go for a little more. 5 minutes?